app/gui/src/controller/searcher.rs

```diff
@@ -1085,24 +1121,25 @@ impl Searcher {
     ///
     /// This is a workaround for Engine bug https://github.com/enso-org/enso/issues/1605.
     //TODO[ao] this is a temporary workaround.
-    fn add_enso_project_entries(libraries_cat_builder: &action::CategoryBuilder) -> FallibleResult {
+    fn add_enso_project_entries(libraries_cat_builder: &action::CategoryBuilder) {
```

Comment:
remove "enso" name from the function pls. We should not encode the product name in the source code in other places than modules and crate names. I know this was not changed here, but as this code is touched, I added this comment for brevity :D

Reply:
The function name reflects the entity in our language. I will fill an issue for engine to remove Enso part from name.